libraries/state_history/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+file(GLOB HEADERS "include/eosio/state-history/*.hpp")
+
+add_library( state_history
+             abi.cpp
+             compression.cpp
+             create_deltas.cpp
+             trace_converter.cpp
+             ${HEADERS}
+           )
+
+target_link_libraries( state_history 
+                       PUBLIC eosio_chain fc chainbase softfloat
+                     )
+
+target_include_directories( state_history
+                            PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" "${CMAKE_CURRENT_SOURCE_DIR}/../wasm-jit/Include"
```

Comment:
It doesn't seem you actually need `"${CMAKE_CURRENT_SOURCE_DIR}/../wasm-jit/Include"` here.

Reply:
removed